prow/cmd/crier/main.go

```diff
@@ -237,18 +237,21 @@ func main() {
 		slackConfig := func(refs *prowapi.Refs) config.SlackReporter {
 			return cfg().SlackReporterConfigs.GetSlackReporter(refs)
 		}
-		if err := secretAgent.Add(o.slackTokenFile); err != nil {
-			logrus.WithError(err).Fatal("could not read slack token")
+		var tokensMap map[string]func() []byte
```

Comment:
You need to initialize the map before you can write to it

Reply:
Nice catch.

Reply:
done